.storybook/main.js

```diff
@@ -1,8 +1,20 @@
+const path = require('path');
+
 module.exports = {
   stories: [
     '../app/components/**/*.stories.@(js|jsx|ts|tsx)',
     '../app/views/**/*.stories.@(js|jsx|ts|tsx)',
     '../app/views/**/**/*.stories.@(js|jsx|ts|tsx)',
   ],
-  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
-};
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials"
+  ]
+  // webpackFinal: async (config, { configType }) => {
+  //   config.resolve.alias = {
+  //     ...config.resolve.alias,
+  //     'fs': path.resolve(__dirname, 'fsMock.js'),
+  //   };
+  //   return config
+  // }
```

Comment:
Why this comment?

Reply:
For reference so I don't forget how to do this. It was irritatingly hard to find a working example.